render.js

```diff
@@ -1,3 +1,17 @@
 "use strict"
 
+// When running in node, we get reference errors on import
+// unless we have window, document, requestAnimationFrame
+try {
+	var isRunningInNode = global.process.version // a good enough proxy
+	if (isRunningInNode){
+		global.window = global.window || undefined
+		global.document = global.document || undefined
+		global.requestAnimationFrame = global.requestAnimationFrame || undefined
+	}
+}
+finally {
+	null
```

Comment:
Remove this line.

Reply:
removing this gives me the `jslint` error 

> 13:9  error  Empty block statement  no-empty

Reply:
We use ESLint, not JSLint. Try it some time - it's pretty awesome. 🙂

Reply:
Sorry, I meant ESLint - it's the failure in CI.